### PR TITLE
Allow minting of zero value for GoldToken and StableToken

### PR DESCRIPTION
**Pull request by nategraf**
## Description

Minting of zero value is currently disallowed in the GoldToken and StableToken contracts. This PR allows minting zero value.

### CODE REVIEW

File: packages/protocol/contracts/common/GoldToken.sol [hunk 1]
```diff
@@ -141,7 +141,11 @@ contract GoldToken is Initializable, CalledByVm, Freezable, IERC20, ICeloToken {
    * @param value The amount of cGLD to mint.
    */
   function mint(address to, uint256 value) external onlyVm returns (bool) {
-    require(value > 0, "mint value must be > 0");
+    require(value >= 0, "mint value must be >= 0");
```
Comment (asaj) [inline packages/protocol/contracts/common/GoldToken.sol]: unnecessary check for unsigned int, please remove

File: packages/protocol/contracts/stability/StableToken.sol [hunk 2]
```diff
@@ -218,7 +218,11 @@ contract StableToken is
    */
   function _mint(address to, uint256 value) private returns (bool) {
     require(to != address(0), "0 is a reserved address");
-    require(value > 0, "mint value must be > 0");
+    require(value >= 0, "mint value must be >= 0");
```
Comment (asaj) [inline packages/protocol/contracts/stability/StableToken.sol]: unnecessary check for unsigned int, please remove

File: packages/protocol/test/stability/stabletoken.ts [hunk 3]
```diff
@@ -146,6 +146,18 @@ contract('StableToken', (accounts: string[]) => {
...
 
+    it('should allow the minting 0 value', async () => {
+      await stableToken.mint(validators, 0, { from: validators })
+      const balance = (await stableToken.balanceOf(validators)).toNumber()
+      assert.equal(balance, 0)
+      const supply = (await stableToken.totalSupply()).toNumber()
+      assert.equal(supply, 0)
+    })
+
+    it('should not allow the minting negative value', async () => {
```
Comment (asaj) [inline packages/protocol/test/stability/stabletoken.ts]: I'm not sure whats' being tested here

**Review by asaj — APPROVED** [review]

File: packages/protocol/contracts/common/GoldToken.sol (hunk 1 above)
Comment (nategraf) [reply packages/protocol/contracts/common/GoldToken.sol]: Ah, right!

File: packages/protocol/test/stability/stabletoken.ts (hunk 3 above)
Comment (nategraf) [reply packages/protocol/test/stability/stabletoken.ts]: Absolutely nothing!

**Pull request merged**